packages/medusa/src/commands/utils/get-migrations.js

```diff
@@ -201,22 +205,17 @@ export const getModuleMigrations = (configModule, isFlagEnabled) => {
 
     const isolatedMigrations = {}
     const moduleMigrations = (mod.migrations ?? [])
-      .map((migrations) => {
-        const all = []
-        for (const migration of Object.values(migrations)) {
-          // TODO: revisit how Modules export their migration entrypoints up/down
-          if (["up", "down"].includes(migration.name)) {
-            isolatedMigrations[migration.name] = migration
-          } else if (
-            typeof migration.featureFlag === "undefined" ||
-            isFlagEnabled(migration.featureFlag)
-          ) {
-            all.push(migration)
-          }
+      .map((migration) => {
```

Comment:
suggestion: Should we move that to a local utility? It seams that this operation is the same as line 183

Reply:
not quite the same. one of them there is a require.

Reply:
Oups missed it 👌